librz/core/cmd_api.c

```diff
@@ -1089,6 +1089,87 @@ static char *get_help(RzCmd *cmd, RzCmdDesc *cd, RzCmdParsedArgs *args, bool use
 	return NULL;
 }
 
+
+static bool child_help_json(RzCmd *cmd, RzCmdDesc *cd, PJ* j) {
```

Comment:
I think when printing in JSON it makes sense to not only return the `args_str`, but also an array of `RzCmdDescArg`, with all the details. It can be much more useful to callers than the string itself.